GramAddict/core/log.py

```diff
@@ -29,9 +32,13 @@ def format(self, record):
 
 
 def configure_logger():
+    log_name = datetime.now().strftime("%Y-%m-%dT%H-%M-%S.%f_run.log")
+    if not os.path.exists("logs"):
+        os.makedirs("logs")
     init_colorama()
     logger = logging.getLogger()  # root logger
-    logger.setLevel(logging.INFO)
+    logger.setLevel(logging.DEBUG)
+    file_handler = RotatingFileHandler(f"logs/{log_name}", mode="a", backupCount=10)
```

Comment:
I think we should specify here the encoding and the max file size (maybe 5 Mb maximum?)

Reply:
Encoding utf-8 avoid those errors when logging weird characters.

Reply:
I did 1Mb. Didn't see this before pushing. It's an arbitrary choice either way. Doesn't matter.